src/PaxfulApi.ts

```diff
@@ -68,9 +62,112 @@ export class PaxfulApi {
      * @param url - Url that should be called at api.paxful.com
      * @param payload - (Optional) Payload of the request
      */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
-    public invoke(url: string, payload?: InvokeBody): Promise<any> {
-        return invoke(url, this.apiConfiguration, this.credentialStorage, payload);
+    public invoke(url: string, payload?: Record<string, unknown> | []): Promise<any> {
+        return invoke((
+            new RequestBuilder(`${process.env.PAXFUL_DATA_HOST}${url}`)
+                .acceptJson()
+                .withMethod("POST")
+                .withFormData(payload)
+        ), this.apiConfiguration, this.credentialStorage);
+    }
+
+    /**
+     * Uploads files on behalf of currently authenticated user,
+     * assumes that payload contains multipart form data.
+     *
+     * @param url - Url that should be called at api.paxful.com
+     * @param payload - (Optional) Payload of the request
+     * @param method - (Optional) Method to use. Default: POST
+     */
+    public upload(url: string, payload: Record<string, unknown> | [], method="POST"): Promise<any> {
+        return invoke((
```

Comment:
For this and all other newly introduced method I was perplexed for some time as to how you're invoking invoke() on PaxfulAPI because it has different signature and accepts string, Record<String, ?>.

Please consider renaming import of "invoke" fn from "commands" file it would have a different name.

Reply:
The `Record` is needed to make sure that it receives any generic object.

Reply:
Clear, will rename invoke() in commands